rssfeed/tests/settings/110.py

```diff
@@ -0,0 +1,54 @@
+DEBUG = True
+
+DATABASES = {
+    "default": {
+        "ENGINE": "django.db.backends.sqlite3",
```

Comment:
Take a glance at my comment in the other settings file about the db engine please.